notebook/agentchat_groupchat_RAG.ipynb

```diff
@@ -117,27 +117,30 @@
     "    description=\"Assistant who has extra content retrieval power for solving difficult problems.\",\n",
     ")\n",
     "\n",
+    "coder_llm_config=llm_config.copy()\n",
```

Comment:
Instead of making a copy which is not obvious from a quick skim of the code, we should create separate `llm_cong` in the constructor, and have them share the same config_list in a separate cell in the beginning. 

E.g.,

```python
AssistantAgent(
  name = "product manager",
  llm_config={"config_list": config_list},
)
```

Reply:
Would you like to address this? @jtoy

Reply:
all fixed!